packages/components/src/card/types.ts

```diff
@@ -90,8 +98,8 @@ type MarginalSubComponentProps = BaseSubComponentProps & {
 	isBorderless?: boolean;
 };
 
-export type HeaderProps = MarginalSubComponentProps;
+export type HeaderProps = MarginalSubComponentProps & SizeableProps;
 
 export type FooterProps = MarginalSubComponentProps & {
 	justify?: CSSProperties[ 'justifyContent' ];
-};
+} & SizeableProps;
```

Comment:
Are these additions necessary? It looks like `SizeableProps` is already included through `BaseSubComponentProps`.